src/theme.js

```diff
@@ -0,0 +1,54 @@
+export const variantSettingsAlert = {
+	primary: {
+		color: "#004085",
+		backgroundColor: "#cce5ff",
+		borderColor: "#b8daff",
+	},
```

Comment:
W mojej ocenie lepiej i wygodniej byłoby zrobić coś na zasadzie:
```
const alert = {
  primary: ...
  secondary: ...
}

...

const theme = {
  alert: alert,
  button: ...
}

export default theme;
```

PS. Oczywiście to spowoduje zmianę odbieranie tych danych w plikach .styled ;)